.github/CONTRIBUTING.md

```diff
@@ -12,53 +11,44 @@ Support KB staff members review issues and pull requests on a regular basis. We
 Support KB works with Magento Community Engineering teams and projects.
 As you contribute PRs, you gain [Contribution Points](../docs/contribution-points.md).
 
-
 ## Get started
 
-
 ![Get started workflow](../docs/img/contributor_start.png)
 
-1. Make sure you have a [GitHub account](https://github.com/signup/free).
-
-    **Note for partners:** Add [2FA](https://devdocs.magento.com/contributor-guide/contributing.html#two-factor) protection when contributing to Magento repositories.
-
-1. [Fork](https://help.github.com/articles/fork-a-repo/) the [Support KB repository](https://github.com/magento/knowledge-base). Remember to [sync your fork](https://help.github.com/articles/syncing-a-fork/) and update branches as needed.
-1. Review the [Support KB guidelines below](#contribution-guidelines).
+* Make sure you have a [GitHub account](https://github.com/signup/free).
+* [Fork](https://help.github.com/articles/fork-a-repo/) the [Support KB repository](https://github.com/magento/knowledge-base). Remember to [sync your fork](https://help.github.com/articles/syncing-a-fork/) and update branches as needed.
+* Review the [Support KB guidelines below](#contribution-guidelines).
 
+>**Note for partners:** Add [2FA](https://devdocs.magento.com/contributor-guide/contributing.html#two-factor) protection when contributing to Magento repositories.
 
 ## Contribute
 
 The following diagram shows the contribution workflow:
 
 ![Contributing workflow](../docs/img/Contributor-PR.png)
 
-
 ### Create a branch
 
-1. Create a new branch from your fork using a name that best describes the work or references a GitHub issue number.
-1. Edit or create markdown (`.md`) files in your branch.
-1. Push your branch to your fork.
+* Create a new branch from your fork using a name that best describes the work or references a GitHub issue number.
+* Edit or create markdown (`.md`) files in your branch.
+* Push your branch to your fork.
 
```

Comment:
Hey @manavluhar , why do you think unordered list is more appropriate here? I think that order of steps is important in this case.

Reply:
Correct ✔, Applied suggested changes. Thank you Oleksandra Marchenko 🙌